File: x5webview/src/main/java/com/web/WebActivity.java
package com.web;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.WindowManager;
import android.widget.Button;
import android.widget.Toast;
import com.example.test_webview_demo.R;
import com.web.widget.X5WebView;

public class WebActivity extends AppCompatActivity {

    private X5WebView webView;
    private Button btnBack;

    public static final String URL_EMPTY_0 = "https://robot-lib-achieve.zuoshouyisheng.com/?app_id=5cc197e8b60c48171066f0e7";
    public static final String URL_EMPTY_1 = "https://robot-lib-achieve.zuoshouyisheng.com/?app_id=5cd3d5cbb60c48343fafe493";

    private String url;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        enableFullScreen(true);
        url = getIntent().getStringExtra("url");
        setContentView(R.layout.activity_web);
        if (getSupportActionBar() != null) {
            getSupportActionBar().hide();
        }
        webView = findViewById(R.id.x5WebView);

        btnBack = findViewById(R.id.btnBack);
        btnBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });

        TbsInitHelper.setListener(new TbsInitHelper.Listener() {
            @Override
            public void onInitComplete() {
                if (url != null && !url.isEmpty()) {
                    webView.loadUrl(url);
                } else {
                    webView.loadUrl(URL_EMPTY_0);
                }
            }

            @Override
            public void onInitError() {
                Toast.makeText(WebActivity.this.getApplicationContext(), "error", Toast.LENGTH_SHORT).show();
            }
        });
    }

    private void enableFullScreen(boolean enable) {
        if (enable) {
            WindowManager.LayoutParams lp = getWindow().getAttributes();
            lp.flags |= WindowManager.LayoutParams.FLAG_FULLSCREEN;
            getWindow().setAttributes(lp);
            getWindow().addFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
        } else {
            WindowManager.LayoutParams attr = getWindow().getAttributes();
            attr.flags &= (~WindowManager.LayoutParams.FLAG_FULLSCREEN);
            getWindow().setAttributes(attr);
            getWindow().clearFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
        }
    }

    @Override
    protected void onDestroy() {
        TbsInitHelper.setListener(null);
        super.onDestroy();
    }
}

File: app/src/main/java/com/afirez/app/ui/WebActivity.kt
package com.afirez.app.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.afirez.app.R
import com.afirez.app.WebFragment

class WebActivity : AppCompatActivity() {

    val url = "http://192.168.200.25:8080/index"
    val url0 = "https://robot-lib-achieve.zuoshouyisheng.com/?app_id=5cc197e8b60c48171066f0e7"
    val url1 = "https://robot-lib-achieve.zuoshouyisheng.com/?app_id=5cd3d5cbb60c48343fafe493"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_web)
        if (supportActionBar != null) {
            supportActionBar?.hide()
        }
        val fm = supportFragmentManager
        val tag = WebFragment::class.java.name
        val fragment = fm.findFragmentByTag(tag)
        if (fragment == null) {
            fm
                .beginTransaction().add(android.R.id.content, WebFragment().apply {
                    arguments = Bundle().apply {
                        val type = intent.getIntExtra("type", 0)
                        val url = if(type == 0) {
                            url0
                        } else {
                            url1
                        }
                        putString("url", url)
                    }
                })
                .commit()
        }
    }
}
